src/connectors/panoptic/panoptic.config.ts

```diff
@@ -24,13 +25,9 @@ export namespace PanopticConfig {
   }
 
   export const config: NetworkConfig = {
-    allowedSlippage: ConfigManagerV2.getInstance().get(
-      'panoptic.allowedSlippage'
-    ),
-    // TODO: Add a getTransactionGasLimitEstimate(unsignedTransactionPayload) here
-    gasLimitEstimate: ConfigManagerV2.getInstance().get(
-      `panoptic.gasLimitEstimate`
-    ),
+    allowedSlippage: ConfigManagerV2.getInstance().get('panoptic.allowedSlippage'),
+    gasFactor: ConfigManagerV2.getInstance().get('panoptic.gasFactor'),
```

Comment:
How about, for these two vars, we go with: "gasLimitCushionFactor", and "absoluteGasLimit", as these are how we're using the two variables

Reply:
Done: 3cb3049508472f435ca7284454ffc9f11717fa25